converters/fromZigbee.js

```diff
@@ -3427,6 +3427,90 @@ const converters = {
             return payload;
         },
     },
+    drayton_device_info: {
```

Comment:
Maybe rename this and the others to `wiser_xxx`, my test unit branded as 'Eberle Wiser' also reports with the exact same database entry.

I looks like both Drayton and Eberle are local rebrands from the Wiser device (which seems to be made by Schnieder)

Reply:
wiser is a good name, as the device model is WV704....

Reply:
Fixed